ocs_ci/deployment/baremetal.py

```diff
@@ -223,23 +224,24 @@ def deploy_prereq(self):
             else:
                 raise RhcosImageNotFound
             # Download metal_bios
-            metal_image_path = (
-                constants.coreos_url_prefix + image_data["metal_bios_url"]
-            )
-            if check_for_rhcos_images(metal_image_path):
-                cmd = (
-                    "wget -O "
-                    f"{self.helper_node_details['bm_path_to_upload']}"
-                    "/rhcos-metal.x86_64.raw.gz "
-                    f"{metal_image_path}"
+            if float_ocp_version <= 4.6:
+                metal_image_path = (
+                    constants.coreos_url_prefix + image_data["metal_bios_url"]
                 )
-                assert self.helper_node_handler.exec_cmd(
-                    cmd=cmd
-                ), "Failed to Download required File"
-            else:
-                raise RhcosImageNotFound
+                if check_for_rhcos_images(metal_image_path):
+                    cmd = (
+                        "wget -O "
+                        f"{self.helper_node_details['bm_path_to_upload']}"
+                        "/rhcos-metal.x86_64.raw.gz "
```

Comment:
can you make this constant at the top of this same file ?

Reply:
Done